modules/hcp-custom-ovn-subnets.adoc

```diff
@@ -0,0 +1,187 @@
++
+For full details about creating a hosted cluster, see "Creating a hosted cluster by using the CLI".
+
+* To configure internal OVN subnets in an existing hosted cluster, enter the following command:
++
+[IMPORTANT]
+====
+When you make this change to an existing hosted cluster, the `ovnkube-node` DaemonSet is rolled out and the OVN components on compute nodes are restarted. During this process, you might experience brief network disruptions.
+====
++
+[source,terminal]
+----
+$ oc patch hostedcluster <hosted_cluster_name> \
+  -n <hosted_control_plane_namespace> \
+  --type=merge \
+  -p '{
+    "spec": {
+      "operatorConfiguration": {
+        "clusterNetworkOperator": {
+          "ovnKubernetesConfig": {
+            "ipv4": {
+              "internalJoinSubnet": "100.99.0.0/16",
+              "internalTransitSwitchSubnet": "100.69.0.0/16"
+            }
+          }
+        }
+      }
+    }
+  }'
+----
++
+where:
+
+`<hosted_cluster_name>`:: Specifies the name of the hosted cluster.
+`<hosted_control_plane_namespace>`:: Specifies the name of the hosted control namespace.
+`ipv4`:: Both subnet fields in this section must be in a valid IPv4 CIDR notation, such as `192.168.1.0/24`. The prefix range is `/0` to `/30`, inclusive. The first octet cannot be 0, and the string length must be 9-18 characters. The subnet fields cannot use the same value. The subnet must be large enough to accommodate one IP address per node in the cluster. When you plan subnet size, consider future cluster growth. If you omit these fields, the default value for the `internalJoinSubnet` field is `100.64.0.0/16`, and the default value for the `internalTransitSwitchSubnet` field is `100.88.0.0/16`.
+
+.Verification
+
+. Verify that the configuration is correct by entering the following command:
++
+[source,terminal]
+----
+$ oc get hostedcluster <hosted_cluster_name> -n <hosted_control_plane_namespace> \
+  -o jsonpath='{.spec.operatorConfiguration.clusterNetworkOperator.ovnKubernetesConfig}' | jq .
+----
++
+.Example output
+[source,terminal]
+----
+{
+  "ipv4": {
+    "internalJoinSubnet": "100.99.0.0/16",
+    "internalTransitSwitchSubnet": "100.69.0.0/16"
+  }
+}
+----
+
+. Check the Network Operator configuration in the hosted cluster by entering the following commands:
++
+[source,terminal]
+----
+$ oc extract secret/<hosted_cluster_name>-admin-kubeconfig -n <hosted_control_plane_namespace> --to=- > hosted-kubeconfig
```

Comment:
the command cannot get hosted-kubeconfig directly, need to oc get secret/<hosted_cluster_name>-admin-kubeconfig -n clusters -o jsonpath='{.data.kubeconfig}' |base64 -d